coderd/notifications/notifier.go

```diff
@@ -195,9 +200,18 @@ func (n *notifier) deliver(ctx context.Context, msg database.AcquireNotification
 
 	ctx, cancel := context.WithTimeout(ctx, n.cfg.DispatchTimeout.Value())
 	defer cancel()
-	logger := n.log.With(slog.F("msg_id", msg.ID), slog.F("method", msg.Method))
+	logger := n.log.With(slog.F("msg_id", msg.ID), slog.F("method", msg.Method), slog.F("attempt", msg.AttemptCount+1))
+
+	if msg.AttemptCount > 0 {
```

Comment:
I take it this is incremented somewhere not visible in this PR?

Reply:
Correct; in the database layer.